src/app/(events)/eventlist/page.tsx

```diff
@@ -25,6 +45,24 @@ const EventListPage = () => {
   const hasResult = !loading && !error && result && result?.length > 0;
   const noResult = !loading && !error && result?.length === 0;
 
+   const { data: session } = useSession();
+
+   const { mutate } = trpc.favoriteEvents.addFavorite.useMutation();
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Don't do it 😤

Reply:
Hi @kei95! It's nice to hear from you here. Thank you for reviewing my code! Would you mind providing more clarity on what you mean??

Reply:
Haha didn't meat to be mean here but I love what you all have made so far! The comment was about the disable eslint comment. It's generally a good idea to avoid doing this or leave context of why you're doing this at really least

Reply:
Got it! Thanks for hopping in and giving me advice!